src/main/java/org/jahia/modules/sam/graphql/GqlHealthCheck.java

```diff
@@ -49,8 +50,10 @@ public GqlProbeStatus getStatus(DataFetchingEnvironment environment) {
 
     @GraphQLField
     @GraphQLDescription("Probes registered in SAM for the requested severity")
-    public List<GqlProbe> getProbes() {
+    public List<GqlProbe> getProbes(@GraphQLName("health") @GraphQLDescription("Return probes matching this status or above") GqlProbeStatus.GqlProbeHealth health) {
+
         return probesRegistry.getProbes().stream()
+                .filter(p -> health == null || p.getStatus().getHealth().ordinal() >= health.ordinal())
```

Comment:
There are two different probe status methods that needs to handled here depending if it needs http context or not.

See implementation of `GqlProbe.getStatus(DataFetchingEnvironment)` that is used in `GqlHealthCheck.getStatus(DataFetchingEnvironment)`